src/__tests__/CompareResults/ResultsView.test.tsx

```diff
@@ -159,7 +159,7 @@ describe('Results View', () => {
     expect(graphTitle).toBe('Runs Density Distribution');
   });
 
-  it('should make blobUrl available when "Download JSON" button is clicked', async () => {
+  it('should make blobUrl available when Download JSON" button is clicked', async () => {
```

Comment:
looks like a typo change, you removed the `"` :-)

Reply:
No idea how that happened, but thanks for catching it! I've fixed it in the current change.